test/unit/color.js

```diff
@@ -88,8 +88,8 @@
     var oColor = new fabric.Color('ffffffff');
     assert.ok(typeof oColor.toHexa === 'function');
     assert.equal(oColor.toHexa(), 'FFFFFFFF');
-    oColor.setSource([255,255,255,0.8]);
-    assert.equal(oColor.toHexa(), 'FFFFFFCC');
+    oColor.setSource([255,255,255,0.1]);
+    assert.equal(oColor.toHexa(), 'FFFFFF1A');
```

Comment:
can i ask you the reason of this change here? i remember this one was here because of weird decimals.

Reply:
It was in my forked version.

Reply:
changed that to original